public/Lab-Files/Lesson03/Assignment/blog-page.html

```diff
@@ -0,0 +1,103 @@
+            Alison, who is the writer of my favorite transplant Texan in Seatle blog, Swears and Sweet Tea.</p>
+    </article>
+    <a href="#top" class="back-top">Back to Top</a>
+    
+    <article class="music-notes">
+        <h2 id="music" class="heading">Music</h2>
+        <h3 class="date">Daily Can't-Get-This-Song-Out-Of-My-Head Playlist</h3>
+        <ol class="small">
+            <li>She's Out of Her Mind - Blink182</li>
+            <li>Heathens - twenty one pilots</li>
+            <li>Code Monkey - Jonathan Coulton (told ya)</li>
+            <li>From God's Perspective - Bo Burnham</li>
+            <li>Harvest Moon - Erin & Evan (yes, I get my own songs stuck in my head sometimes too)</li>
+        </ol>
+        
+        <h3>SoundCloud Music:</h3>
+        <a href="https://soundcloud.com/erinandevanmusic" class="mlinks"><img src="Image/ErinandEvan.jpg" alt="headshot of Erin & Evan looking through round glass washer door at Laundromat"/></a>
+        <a href="https://soundcloud.com/erin-krummenacher" class="mlinks"><img src="Image/ErinArtsyShot.jpeg" alt="Erin standing outside in black dress looking down"/></a><br>
+    </article> 
+    
+    <div class="back-top">
+    <a href="#top">Back to Top</a>
+    </div>
+    
+    <article class="aboutme">
+        <h2 id="about-me" class="heading">About Me aka The Musical Nerd</h2>
+            <p class="small">Hello, my name is Erin and I am your run-of-the-mill musically talented nerd. Texas born and raised<br>
+            on a farm is where I did not spend most of my days. Try more like suburbs of Houston. Went to college up <br>
+            in almost Oklahoma North Texas at Austin College (Go Roos!) and against all judgement studies music. <br>
+            Now don't get me wrong, I loved getting my degree in Music, but I do wish I had double majored in something <br>
+            relatively marketable. And by marketable, I mean for more than $9/hr that wasn't a call center. Hindsight is <br>
+            20/20. Anyways...met my husband at college, got married a week after graduation (don't try this at home kids), <br>
+            and promptly moved away to New Hampshire. All the while, I am a big BIG fan of shows like Doctor Who, Sherlock, <br>
+            and Agents of Shield, and have become ingrained in the fandom life in my spare time (aka 'Nerd' part of me). I am<br>
+            half of a folk duo band called 'Erin & Evan' where Evan and I sing, play guitar (Evan), play mandolin (me), and <br>
+            make/write generally good music. We are working on an album of originals that we have written and recorded with the <br>
+            help of our good friend, Reo. I am positive you will hear all about the progress of this in the above blog posts. <br>
+            Enough about me, now go read things that I thought of and wrote. DFTBA!</p>
+        
+    </article>
+    <a href="#top" class="back-top">Back to Top</a>
+    </div>
+</body>
+</html>
```

Comment:
I like the way you included anchor tags and made your page more interactive. It's awesome. 
Good job at validating the code too.